CHANGES.rst

```diff
@@ -4,6 +4,13 @@
 New Features
 ^^^^^^^^^^^^
 
+- Added 'self_profile' option for HorneExtract. If The ``self_profile`` option

```

Comment:
i think `interpolated_profile` is a better name and more accurately descriptive of it.

Reply:
done